### PR TITLE
[DXHALTEST] Fix some typos

### PR DESCRIPTION
**Pull request by diakin**
## Purpose

fixed some typos

### CODE REVIEW

**Comment by HBelusca** [comment]
i.e. translate Greatlordish into proper english :rofl:

**Comment by jeditobe** [comment]
GreatLordish is a very distinct language , and sometimes can take a few runs through a translator.

**Comment by diakin** [comment]
omg.. Magnus Olsen aka GreatLord...

File: modules/rostests/tests/DxHalTest/main.c [hunk 1]
```diff
@@ -99,15 +99,15 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
...
 	printf("This apps showing how to start up directx draw/d3d interface and some other as well\n");
-	printf("This code have been releae to some close applactons with my premtions, if any company\n");
-    printf("want use part or whole code, you need contact the orginal author to ask for premtions\n");
-    printf("This code are release under alot of diffent licen\n");
-    printf("All GPL and LGPL project have right use and studing this code.\n");
-    printf("This code maybe need more comment to known how stuff working and maybe looking bit mesy\n");
-	printf("Bestreagds Magnus Olsen magnus@greatlord.com or greatlord@reactos.org\n");
+	printf("This code have been released to some close applications with my promotions, if any company\n");
+    printf("want use part or whole code, you need contact the original author to ask for permission\n");
+    printf("This code are released under a lot of different licenses\n");
```
Comment (gonzoMD) [inline modules/rostests/tests/DxHalTest/main.c:104]: this Code *is* released...

File: modules/rostests/tests/DxHalTest/main.c [hunk 2]
```diff
@@ -99,15 +99,15 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
     DdResetVisrgn            = (BOOL (APIENTRY*)(LPDDRAWI_DDRAWSURFACE_LCL, HWND))GetProcAddress(lib, "GdiEntry6");
 
 	printf("This apps showing how to start up directx draw/d3d interface and some other as well\n");
-	printf("This code have been releae to some close applactons with my premtions, if any company\n");
-    printf("want use part or whole code, you need contact the orginal author to ask for premtions\n");
-    printf("This code are release under alot of diffent licen\n");
-    printf("All GPL and LGPL project have right use and studing this code.\n");
-    printf("This code maybe need more comment to known how stuff working and maybe looking bit mesy\n");
-	printf("Bestreagds Magnus Olsen magnus@greatlord.com or greatlord@reactos.org\n");
+	printf("This code have been released to some close applications with my promotions, if any company\n");
```
Comment (gonzoMD) [inline modules/rostests/tests/DxHalTest/main.c:102]: premtions seems to mean permission. (same as in line 103)

Comment (gonzoMD) [reply modules/rostests/tests/DxHalTest/main.c:102]: s/have/has

**Review by gonzoMD — COMMENTED** [review]
some quick first-view-comments by me.

File: modules/rostests/tests/DxHalTest/main.c (hunk 2 above)
Comment (diakin) [reply modules/rostests/tests/DxHalTest/main.c]: >s/have/has
Where is it?

File: modules/rostests/tests/DxHalTest/main.c (hunk 1 above)
Comment (diakin) [reply modules/rostests/tests/DxHalTest/main.c]: Changed

File: modules/rostests/tests/DxHalTest/main.c (hunk 2 above)
Comment (diakin) [reply modules/rostests/tests/DxHalTest/main.c]: "my promotions" changed to "my permission"

Comment (gonzoMD) [reply modules/rostests/tests/DxHalTest/main.c]: > > s/have/has
> > Where is it?

This code have been - - > this Code has been

File: modules/rostests/tests/DxHalTest/main.c [hunk 3]
```diff
@@ -99,15 +99,15 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
...
-	printf("This code have been releae to some close applactons with my premtions, if any company\n");
-    printf("want use part or whole code, you need contact the orginal author to ask for premtions\n");
-    printf("This code are release under alot of diffent licen\n");
-    printf("All GPL and LGPL project have right use and studing this code.\n");
-    printf("This code maybe need more comment to known how stuff working and maybe looking bit mesy\n");
-	printf("Bestreagds Magnus Olsen magnus@greatlord.com or greatlord@reactos.org\n");
+	printf("This code has been released to some close applications with my permission, if any company\n");
+    printf("want use part or whole code, you need contact the original author to ask for permission\n");
+    printf("This code is released under a lot of different licenses\n");
+    printf("All GPL and LGPL projects has the right to use and study this code.\n");
```
Comment (JoachimHenze) [inline modules/rostests/tests/DxHalTest/main.c:105]:
```suggestion
    printf("All GPL and LGPL projects have the right to use and study this code.\n");
```

File: modules/rostests/tests/DxHalTest/main.c [hunk 4]
```diff
@@ -99,15 +99,15 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
...
+    printf("This code maybe need more comments to know how stuff working and maybe looking а bit messy\n");
```
Comment (JoachimHenze) [inline modules/rostests/tests/DxHalTest/main.c:106]:
```suggestion
    printf("This code maybe need more comments to know how stuff works.\n");
```

File: modules/rostests/tests/DxHalTest/main.c [hunk 5]
```diff
@@ -99,15 +99,15 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
...
+	printf("Best regards Magnus Olsen magnus@greatlord.com or greatlord@reactos.org\n");
     printf("Copyright 2006 by Magnus Olsen\n\n");
     printf("This demo showing how to  start dx draw hal and create a primary surface,\n");
```
Comment (JoachimHenze) [inline modules/rostests/tests/DxHalTest/main.c:109]:
```suggestion
    printf("This demo showing how to start dx draw hal and create a primary surface,\n");
```

File: modules/rostests/tests/DxHalTest/main.c [hunk 6]
```diff
@@ -99,15 +99,15 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
...
-	printf("and a overlay sufrace and blt to the primary surface\n");
+	printf("and a overlay surface and blt to the primary surface\n");
```
Comment (JoachimHenze) [inline modules/rostests/tests/DxHalTest/main.c:110]:
```suggestion
	printf("and an overlay surface and blt to the primary surface\n");
```

File: modules/rostests/tests/DxHalTest/main.c [hunk 7]
```diff
@@ -99,15 +99,15 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
     DdResetVisrgn            = (BOOL (APIENTRY*)(LPDDRAWI_DDRAWSURFACE_LCL, HWND))GetProcAddress(lib, "GdiEntry6");
 
 	printf("This apps showing how to start up directx draw/d3d interface and some other as well\n");
```
Comment (JoachimHenze) [inline modules/rostests/tests/DxHalTest/main.c:101]:
```suggestion
	printf("This app shows how to start up directx draw/d3d interface and some others.\n");
```

**Comment by diakin** [comment]
Need I do something else to add changes to master branch reactos/reactos/tree/master/modules/rostests/tests/DxHalTest/main.c
?

**Review by binarymaster — APPROVED** [review]

File: modules/rostests/tests/DxHalTest/main.c [hunk 8]
```diff
@@ -98,16 +98,16 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
...
-	printf("This apps showing how to start up directx draw/d3d interface and some other as well\n");
-	printf("This code have been releae to some close applactons with my premtions, if any company\n");
-    printf("want use part or whole code, you need contact the orginal author to ask for premtions\n");
-    printf("This code are release under alot of diffent licen\n");
-    printf("All GPL and LGPL project have right use and studing this code.\n");
-    printf("This code maybe need more comment to known how stuff working and maybe looking bit mesy\n");
-	printf("Bestreagds Magnus Olsen magnus@greatlord.com or greatlord@reactos.org\n");
+	printf("This app shows how to start up directx draw/d3d interface and some others.\n");
+	printf("This code has been released to some close applications with my permission, if any company\n");
+    printf("want use part or whole code, you need contact the original author to ask for permission\n");
```
Comment (HBelusca) [inline modules/rostests/tests/DxHalTest/main.c:103]:
```suggestion
	printf("This code has been released to some closed applications with my permission. If any company\n");
    printf("want to use parts or whole code, you need to contact the original author to ask for permission.\n");
```

File: modules/rostests/tests/DxHalTest/main.c [hunk 9]
```diff
@@ -98,16 +98,16 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
...
+    printf("This code is released under a lot of different licenses\n");
```
Comment (HBelusca) [inline modules/rostests/tests/DxHalTest/main.c:104]:
```suggestion
    printf("This code is released under a lot of different licenses.\n");
```

File: modules/rostests/tests/DxHalTest/main.c [hunk 10]
```diff
@@ -98,16 +98,16 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
...
+    printf("All GPL and LGPL projects have the right to use and study this code.\n");
+    printf("This code maybe need more comments to know how stuff works.\n");
```
Comment (HBelusca) [inline modules/rostests/tests/DxHalTest/main.c:106]:
```suggestion
    printf("This code may require more comments to know how stuff works.\n");
```

File: modules/rostests/tests/DxHalTest/main.c [hunk 11]
```diff
@@ -98,16 +98,16 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
...
+	printf("Best regards Magnus Olsen magnus@greatlord.com or greatlord@reactos.org\n");
     printf("Copyright 2006 by Magnus Olsen\n\n");
-    printf("This demo showing how to  start dx draw hal and create a primary surface,\n");
-	printf("and a overlay sufrace and blt to the primary surface\n");
+    printf("This demo showing how to start dx draw hal and create a primary surface,\n");
```
Comment (HBelusca) [inline modules/rostests/tests/DxHalTest/main.c:109]:
```suggestion
    printf("This demo shows how to start dx draw hal and create a primary surface,\n");
```

File: modules/rostests/tests/DxHalTest/main.c [hunk 12]
```diff
@@ -98,16 +98,16 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
...
+	printf("and an overlay surface and blt to the primary surface\n");
```
Comment (HBelusca) [inline modules/rostests/tests/DxHalTest/main.c:110]:
```suggestion
	printf("and an overlay surface and blt to the primary surface.\n");
```

File: modules/rostests/tests/DxHalTest/main.c [hunk 13]
```diff
@@ -208,10 +208,10 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
 
 	mDDrawGlobal.dwRefCnt = 1; //addref / remove ref
 
-    // Do not relase HDC it have been map in kernel mode
+    // Do not release HDC it has been map in kernel mode
```
Comment (HBelusca) [inline modules/rostests/tests/DxHalTest/main.c:211]:
```suggestion
    // Do not release HDC, it has been mapped in kernel mode
```

File: modules/rostests/tests/DxHalTest/main.c [hunk 14]
```diff
@@ -208,10 +208,10 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
...
     // DeleteDC(hdc);
 
-    /* we need reanable it if screen res have changes, and some bad drv need be reanble very few
+    /* we need reenable it if screen res has changes, and some bad drv need be reenable very few
 	   to contiune */
```
Comment (HBelusca) [inline modules/rostests/tests/DxHalTest/main.c:215]:
```suggestion
    /* we need to reenable it if the screen resolution has changed, and some
       bad driver needs to be reenabled infrequently to continue */
```

File: modules/rostests/tests/DxHalTest/main.c [hunk 15]
```diff
@@ -328,7 +328,7 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
       Copy over from HalInfo to DirectDrawGlobal
    */
 
-  // this is wrong, cDriverName need be in ASC code not UNICODE
+  // this is wrong, cDriverName need be in ASCII code not UNICODE
```
Comment (HBelusca) [inline modules/rostests/tests/DxHalTest/main.c:331]:
```suggestion
  // this is wrong, cDriverName needs to be in ASCII not UNICODE
```

**Review by HBelusca — COMMENTED** [review]
random extra fixes.
But let's remember the comments were written in Greatlordish, so....

File: modules/rostests/tests/DxHalTest/main.c [hunk 16]
```diff
@@ -98,16 +98,16 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
...
-    printf("All GPL and LGPL project have right use and studing this code.\n");
-    printf("This code maybe need more comment to known how stuff working and maybe looking bit mesy\n");
-	printf("Bestreagds Magnus Olsen magnus@greatlord.com or greatlord@reactos.org\n");
+	printf("This app shows how to start up directx draw/d3d interface and some others.\n");
+	printf("This code has been released to some closed applications with my permission. If any company\n");
+    printf("want to use parts or whole code, you need to contact the original author to ask for permission.\n");
+    printf("This code is released under a lot of different licenses.\n");
+    printf("All GPL and LGPL projects have the right to use and study this code.\n");
+    printf("This code may require more comments to know how stuff works.\n");
+	printf("Best regards Magnus Olsen magnus@greatlord.com or greatlord@reactos.org\n");
```
Comment (cbialorucki) [inline modules/rostests/tests/DxHalTest/main.c:107]:
```suggestion
	printf("Best regards, Magnus Olsen magnus@greatlord.com or greatlord@reactos.org\n");
```

File: modules/rostests/tests/DxHalTest/main.c [hunk 17]
```diff
@@ -98,16 +98,16 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
...
-	printf("This apps showing how to start up directx draw/d3d interface and some other as well\n");
-	printf("This code have been releae to some close applactons with my premtions, if any company\n");
-    printf("want use part or whole code, you need contact the orginal author to ask for premtions\n");
-    printf("This code are release under alot of diffent licen\n");
-    printf("All GPL and LGPL project have right use and studing this code.\n");
-    printf("This code maybe need more comment to known how stuff working and maybe looking bit mesy\n");
-	printf("Bestreagds Magnus Olsen magnus@greatlord.com or greatlord@reactos.org\n");
+	printf("This app shows how to start up directx draw/d3d interface and some others.\n");
+	printf("This code has been released to some closed applications with my permission. If any company\n");
+    printf("want to use parts or whole code, you need to contact the original author to ask for permission.\n");
```
Comment (cbialorucki) [inline modules/rostests/tests/DxHalTest/main.c:103]:
```suggestion
    printf("wants to use this code, you must contact the original author to ask for permission.\n");
```

File: modules/rostests/tests/DxHalTest/main.c [hunk 18]
```diff
@@ -98,16 +98,16 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
...
+    printf("This code is released under a lot of different licenses.\n");
```
Comment (cbialorucki) [inline modules/rostests/tests/DxHalTest/main.c:104]:
```suggestion
    printf("This code is released under many different licenses.\n");
```

File: modules/rostests/tests/DxHalTest/main.c [hunk 19]
```diff
@@ -98,16 +98,16 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
...
     DdResetVisrgn            = (BOOL (APIENTRY*)(LPDDRAWI_DDRAWSURFACE_LCL, HWND))GetProcAddress(lib, "GdiEntry6");
 
-	printf("This apps showing how to start up directx draw/d3d interface and some other as well\n");
-	printf("This code have been releae to some close applactons with my premtions, if any company\n");
-    printf("want use part or whole code, you need contact the orginal author to ask for premtions\n");
-    printf("This code are release under alot of diffent licen\n");
-    printf("All GPL and LGPL project have right use and studing this code.\n");
-    printf("This code maybe need more comment to known how stuff working and maybe looking bit mesy\n");
-	printf("Bestreagds Magnus Olsen magnus@greatlord.com or greatlord@reactos.org\n");
+	printf("This app shows how to start up directx draw/d3d interface and some others.\n");
```
Comment (cbialorucki) [inline modules/rostests/tests/DxHalTest/main.c:101]:
```suggestion
	printf("This app shows how to start up a DirectDraw/Direct3D interface and some others.\n");
```

File: modules/rostests/tests/DxHalTest/main.c [hunk 20]
```diff
@@ -98,16 +98,16 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
...
+	printf("This code has been released to some closed applications with my permission. If any company\n");
+    printf("want to use parts or whole code, you need to contact the original author to ask for permission.\n");
+    printf("This code is released under a lot of different licenses.\n");
+    printf("All GPL and LGPL projects have the right to use and study this code.\n");
+    printf("This code may require more comments to know how stuff works.\n");
+	printf("Best regards Magnus Olsen magnus@greatlord.com or greatlord@reactos.org\n");
     printf("Copyright 2006 by Magnus Olsen\n\n");
-    printf("This demo showing how to  start dx draw hal and create a primary surface,\n");
-	printf("and a overlay sufrace and blt to the primary surface\n");
+    printf("This demo shows how to start dx draw hal and create a primary surface,\n");
```
Comment (cbialorucki) [inline modules/rostests/tests/DxHalTest/main.c:109]:
```suggestion
    printf("This demo shows how to start DirectDraw, creating a primary surface\n");
```

File: modules/rostests/tests/DxHalTest/main.c [hunk 21]
```diff
@@ -98,16 +98,16 @@ int WINAPI WinMain (HINSTANCE hInst, HINSTANCE hPrevInst,
...
+	printf("and an overlay surface and blt to the primary surface.\n");
```
Comment (cbialorucki) [inline modules/rostests/tests/DxHalTest/main.c:110]:
```suggestion
	printf("and an overlay surface, and a blt to the primary surface.\n");
```

**Review by JoachimHenze — APPROVED** [review]

**Pull request merged**